site/docs/v1/tech/connectors/generic-connector.adoc

```diff
@@ -8,17 +8,80 @@ description: Connect to an arbitrary user data source over HTTP
 
 == Overview
 
+* <<Configuration>>
+* <<Using the Generic Connector>>
+* <<Security>>
+
 Generic Connectors allow you to authenticate users against or migrate them from any user datasource accessible over HTTP or TLS.
 
-image::connectors/generic-connector-create-screen.png[The Generic Connectors creation page.,width=1200,role=shadowed]
+
+=== Configuration
+
+image::connectors/generic-connector.png[The Generic Connector creation page.,width=1200,role=shadowed]
```

Comment:
Add one additional role for `bottom-cropped` You can see the LDAP one for reference.